tests/src/python/test_qgsserver.py

```diff
@@ -1164,18 +1319,14 @@ def _img_diff_error(self, response, headers, image, max_diff=10, max_size_diff=Q
         test, report = self._img_diff(response, image, max_diff, max_size_diff)
 
         with open(os.path.join(tempfile.gettempdir(), image + "_result.png"), "rb") as rendered_file:
-            encoded_rendered_file = base64.b64encode(rendered_file.read())
```

Comment:
Why remove this?
This is the only way we have to reconstruct the failing image when the test is failing on Travis, if you want to make it conditional, there is an env var which is only set by travis: `TRAVIS=true`